server-side/email-service/app/env

```diff
@@ -0,0 +1,3 @@
+EMAIL_USER=prwyyqtsly@gmail.com
```

Comment:
please create gmail account with meaningful name, such as timeOut, timeOutExtension or something similar

Reply:
spoke to sari about it and decided to leave it as is